Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -137,7 +137,7 @@ static LocalFrame* frameWithSelection(Page* page)
 
 void FindController::updateFindUIAfterPageScroll(bool found, const String& string, OptionSet<FindOptions> options, unsigned maxMatchCount, DidWrap didWrap, FindUIOriginator originator)
 {
-    auto* selectedFrame = frameWithSelection(m_webPage->corePage());
+    RefPtr selectedFrame = frameWithSelection(m_webPage->corePage());
```

Comment:
Maybe `frameWithSelection` should just return `RefPtr`

Reply:
Sure, we can do that.